datadog/dogshell/common.py

```diff
@@ -17,16 +17,14 @@ def print_err(msg):
 
 def report_errors(res):
     if 'errors' in res:
-        for e in res['errors']:
-            print_err('ERROR: ' + e)
+        print_err('ERROR: ' + res['errors'])
```

Comment:
`res["errors"]` can actually be a list: e.g. `{'errors': ['No dashboard matches that dash_id.']}`
let's check for the type first, then iterate or not